python/pyspark/sql/connect/client.py

```diff
@@ -772,6 +772,7 @@ def __exit__(
         if isinstance(exc_val, BaseException):
             # Swallow the exception.
             if self._can_retry(exc_val):
+                logger.debug(f"Server responded with retryable error: {exc_val}")
```

Comment:
Shall we also mention that we're trying from this error? The message itself looks like it has to be either error or warning.

Reply:
If the service is unreachable for a moment and if the retry is going to fix it, it's not an error.
It could be a warn but that'll pollute real warnings. I think it's debug.